api/graphql/index.js

```diff
@@ -245,6 +246,8 @@ export function makeMutations (sessionId, userId, isAdmin) {
 
     createTopic: (root, { topicName, groupId, isDefault, isSubscribing }) => createTopic(userId, topicName, groupId, isDefault, isSubscribing),
 
+    deactivateMe: (root, { sessionId, userId }) => deactivateUser({ sessionId, userId }),
```

Comment:
shouldnt need to pass in userId because for deactivateMe we always want to deactivate the current user